GOVERNANCE.md

```diff
@@ -0,0 +1,150 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the
+`zarr` project, to clarify how decisions are made and how the various
+elements of our community interact.
+
+This is a consensus-based community project. Anyone with an interest in the
+project can join the community, contribute to the project design, and
+participate in the decision making process. This document describes how that
+participation takes place, how to find consensus, and how deadlocks are
+resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The zarr community consists of anyone using or working with the project
```

Comment:
```suggestion
The Zarr community consists of anyone using or working with the project
```

Reply:
Happy to accept this, but we probably should write down explicitly the expected capitalization somewhere. I feel like I've seen both.